unofficial/c511027665.lua

```diff
@@ -0,0 +1,40 @@
+--Abyss Performance
+--Made by Beetron-1 Beetletop
+local s,id=GetID()
+function s.initial_effect(c)
+	--chain attack
+	local e1=Effect.CreateEffect(c)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetCode(EVENT_BATTLE_DESTROYING)
+	e1:SetCondition(s.condition)
+	e1:SetTarget(s.target)
+	e1:SetOperation(s.activate)
+	c:RegisterEffect(e1)
+end
+function s.condition(e,tp,eg,ep,ev,re,r,rp)
+	local tc=eg:GetFirst()
+	local bc=tc:GetBattleTarget()
+	return tc:IsRelateToBattle() and tc:IsStatus(STATUS_OPPO_BATTLE) and tc:IsControler(tp) and tc:IsSetCard(0x10ec)
+		and bc:IsLocation(LOCATION_GRAVE) and bc:IsReason(REASON_BATTLE)
+end
+function s.target(e,tp,eg,ep,ev,re,r,rp,chk)
+	local tg=eg:GetFirst()
+	if chk==0 then return tg:IsOnField() end
+	Duel.SetTargetCard(tg)
+end
+function s.activate(e,tp,eg,ep,ev,re,r,rp)
+	local tc=Duel.GetFirstTarget()
+	if tc:IsFacedown() or not tc:IsRelateToEffect(e) or tc:IsControler(1-tp) or tc:IsImmuneToEffect(e) then return end
+		local e1=Effect.CreateEffect(tc)
+    	e1:SetType(EFFECT_TYPE_SINGLE)
+		e1:SetCode(EFFECT_SET_ATTACK_FINAL)
+		e1:SetValue(s.atkval)

```

Comment:
Should directly set its current ATK divided by 2 as the effect's value, otherwise it'll get recalculated if its ATK changes later.